.editorconfig

```diff
@@ -0,0 +1,25 @@
+# This file is for unifying the coding style for different editors and IDEs
+# It is based on https://core.trac.wordpress.org/browser/trunk/.editorconfig
+# See editorconfig.org for more information
+
+# WordPress Coding Standards
+# https://make.wordpress.org/core/handbook/coding-standards/
```

Comment:
Thoughts on adding a note stating that this is a recommendation but not a requirement and that customers are welcome to modify / adapt / remove as needed based on their team's needs.

Reply:
Can do, but this can also be covered in the public docs, which this file could link to instead?

Reply:
Yep, that works 👍